README.md

```diff
@@ -20,14 +20,14 @@ will result in the following diff:
 ```
 
 Here the tuples `(1, 2), ("x", "y"), (None, True), (1, None)` represent the substitions that happened.
-`None` implies that a object was either removed or added.
+`None` implies that an object was either removed or added.
 
 If there is no diff (e.g.: `diff({ "a": 1 }, { "a": 1 })`) the result will be `None`. In this sense,
 the library can also be used to compare two json-compatible dictionaries.
 
 ---
 
-If a object of any different type is used anywhere in a dict/list/..., a `ValidationException` will be raised.
+If an object of any other type is used in a dict, list, ..., a `ValidationException` will be raised.
 
 ## Exports
```

Comment:
This part I don't fully understand why it is part of the README?

Reply:
I think it doesn't hurt.